.github/workflows/sync_default_branch.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync Default Branch with Target Branch
+
+on:
+  workflow_call:
+    inputs:
+      target_branch:
+        required: false
+        type: string
+        default: storybook-site
+    secrets:
+      caller_github_token:
+        required: true
+
+jobs:
+  sync-branches:
+    runs-on: ubuntu-latest
+    name: Sync branches
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Sync ${{ github.event.repository.default_branch }} with ${{ inputs.target_branch }}
```

Comment:
Is `github.event.repository.default_branch` a built in value to all repos?

Reply:
yes, a default branch is always present in a repo so this field will always be populated